### DISCUSSION TITLE
do not log .xpd file

### DISCUSSION THREAD

**Discussion by assuntaciarlo** [expyriment/expyriment#216] [General]
Hello,
I was wondering if there's a way to avoid saving the .xpd file since I'm using a custom logger.

Many thanks,
Assunta

**Comment by fladd** [comment]
Hi,

sorry to only respond now, I somehow didn't get notified about this discussion. I am answering anyway, for future reference.

You could decide to not start the experiment (i.e. call `control.start`), since this is the point where the data file is created. You would then, however, also need to take care of setting the subject id yourself.
